festim/exports/derived_quantities/average_volume.py

```diff
@@ -3,9 +3,38 @@
 
 
 class AverageVolume(VolumeQuantity):
+    """
+    Computes the average value of a field in a given volume
```

Comment:
We should add the formula like in AverageSurface but with dx (I maybe got mixed up in my previous review)